service/k8sclient/k8client.go

```diff
@@ -636,3 +573,29 @@ func (c *K8Client) setComputeResources(res *ComputeResources) *resourcesSpec {
 	}
 	return r
 }
+
+func (c *K8Client) getDiskSize(volumeSpec *common.VolumeSpec) int64 {
+	if volumeSpec == nil || volumeSpec.PersistentVolumeClaim == nil {
+		return 0
+	}
+	quantity, ok := volumeSpec.PersistentVolumeClaim.Resources.Requests[common.ResourceStorage]
+	if !ok {
+		return 0
```

Comment:
Looks contradictory that in resonance DiskSize can be zero and in the request cannot 
https://github.com/percona-platform/dbaas-api/pull/62/files#diff-3af72cef13d64190ae8c2b265df6707de29a26e75bfd5f09d98870e9effeb3eaR51

Should disc size be optional? 

And maybe the minimum should be at least 300MB. As when MySQL just start it require space for standard and temporary files:

```
48M /data/mysql/ib_logfile0
48M /data/mysql/ib_logfile1
192K /data/mysql/#ib_16384_0.dblwr
8.2M /data/mysql/#ib_16384_1.dblwr
41M /data/mysql/mysql.ibd
14M /data/mysql/undo_001
15M /data/mysql/undo_002
12M /data/mysql/ibtmp1
 ```

Reply:
no, it should not be optional.
at the moment we don't validate any inputs, we will do it later. 
in UI minimum value is 1GB AFAIK.

Reply:
FWIW, the minimal value of 1GB would be the sane minimal value in API too.

We have this problem with zero values in general (zero value in API – not set, not known, forgotten, or really set to zero?), so we can forbid zero value for disk size everywhere, that would be nice

Reply:
it's in response in case we have some db clusters with not set volume values or configured another way